templates/vue/src/components/modals/NodeModal/forms/ConditionsForm.vue

```diff
@@ -9,6 +9,11 @@
         logged in.
       </b-form-invalid-feedback>
     </b-form-group>
+    <b-form-group v-if="lock" class="mb-3">
+      <b-form-checkbox v-model="hideWhenLocked">
+        Also hide this node when locked
```

Comment:
Rather than a checkbox, can we have a toggle on the right side of the "Prevent access..." with 2 options: "Grey out" and "Hide"